test/Driver/driver-compile.swift

```diff
@@ -45,7 +45,9 @@
 
 // RUN: %swiftc_driver -driver-print-jobs -c -target x86_64-apple-macosx10.9 %s %S/../Inputs/empty.swift -module-name main -driver-filelist-threshold=0 2>&1 | %FileCheck -check-prefix=FILELIST %s
 
-// RUN: %empty-directory(%t)/DISTINCTIVE-PATH/usr/bin/
+// RUN: %empty-directory(%t/DISTINCTIVE-PATH/usr/bin)
+// RUN: %empty-directory(%t/DISTINCTIVE-PATH/usr/share/swift/diagnostics)
+// RUN: cp -a %S/Inputs/diagnostics/. %t/DISTINCTIVE-PATH/usr/share/swift/diagnostics
```

Comment:
I had to copy the diagnostics folder in order to make the tests that moves the swift-driver work... not sure if that's the right approach.
cc @xedin